ABOUT.md

```diff
@@ -0,0 +1,47 @@
+
+# About MineColonies 
+MineColonies is an interactive Town building mod that allows you to create your own thriving Town within Minecraft. It depicts real-life scenarios by providing you with different options for you to craft in your chest, to build your own Town  and enhance your gaming experience. Featuring many NPC workers such as: Builders, Crafters, Farmers, Fishermen, Guards, Barracks, Miners, Smeltery, Baker, Cook/Restuarant, Deliveryman, Animal Herders and many more in development. As well as a specialized buildings such as: Warehouse, Citizen Huts and a Townhall. MineColonies gives a player the ability to create a colony rich and unique as every player.
```

Comment:
I would correct: "... different options for you to craft ~~in your chest,~~ to build your own Town... " 
to: "... many craftable options for you to build your own Town... "
As crafting does not happen in any chest in game.
Also: "MineColonies gives ~~a player~~ the ability to create a colony rich and unique as every player. ""
to: "MineColonies gives **you** the ability to create a colony **as** rich and unique as every player. "

Reply:
I'll settle for “...different crafty options to build your own Town... ” 
There is “crafty” but no “craftable”. 
Thanks anyways.

Reply:
What do you mean by: "There is “crafty” but no “craftable”." 
Is English not your main language? Are you using a translation service?
Craftable is a common English word used in MANY real world situations and definitely a MUST in Minecraft in general.

Reply:
Forgive my ignorance. I was carried away by the proofreading. 
Yes, English is my main language and No, I'm not using any translation service. 
I sincerely apologize. The correction has been made.